rotkehlchen/tests/api/test_balances.py

```diff
@@ -138,27 +138,30 @@ def test_query_all_balances(
     rotki = rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen
     rotki.chain_manager.cache_ttl_secs = 0
     setup = setup_balances(rotki, ethereum_accounts, btc_accounts)
-    # Test all balances request by requesting to not save the data
+    # Test that all balances request saves data on a fresh account
     with ExitStack() as stack:
         setup.enter_all_patches(stack)
         response = requests.get(
             api_url_for(
                 rotkehlchen_api_server_with_exchanges,
                 "allbalancesresource",
-            ), json={'save_data': False},
+            ),
         )
 
     assert_proper_response(response)
     json_data = response.json()
     assert_all_balances(
         data=json_data,
         db=rotki.data.db,
-        expected_data_in_db=False,
+        expected_data_in_db=True,
         setup=setup,
     )
 
-    # now do the same but save the data in the DB and test it works
-    # Omit the argument to test that default value of save_data is True
+    # rotki = rotkehlchen_api_server.rest_api.rotkehlchen
```

Comment:
you have commented out code here @isidorosp

Reply:
Fixed